### DISCUSSION TITLE
UseWindowsForms=true disable console

### DISCUSSION THREAD

**Discussion by kant2002** [dotnet/winforms#5042] [Q&A] [answered]
1. Create new console program
2. Set net6.0-windows/net5.0-windows
3. Set UseWindowsForms=true

Now Console completely disabled, even if `<OutputType>Exe</OutputType>`.

**Comment by RussKie** [comment, marked as answer]
This is by design. Please refer to the breaking docs for the workaround: https://docs.microsoft.com/dotnet/core/compatibility/sdk/5.0/automatically-infer-winexe-output-type#recommended-action

**Reply by kant2002** [reply]
Was not aware about that. I suspect something like that, but do not know where to look. Thanks!
